### PR TITLE
Adds plotting options to voronoi_plot_2d

### PR DESCRIPTION
**Pull request by pwolfram**
Simple (minor) additions to plotting routine for additional functionality that is useful for plotting unstructured grids.

### CODE REVIEW

**Comment by pv** [comment]
- The docstring should be updated with the new options
- The example given in the docstring doesn't appear to work: https://travis-ci.org/scipy/scipy/jobs/108433882#L3387

File: scipy/spatial/_plotutils.py [hunk 1]
```diff
@@ -144,19 +144,27 @@ def voronoi_plot_2d(vor, ax=None, **kw):
...
 
-    ax.plot(vor.points[:,0], vor.points[:,1], '.')
+    if kw.get('show_points', True):
+        ax.plot(vor.points[:,0], vor.points[:,1], '.')
     if kw.get('show_vertices', True):
         ax.plot(vor.vertices[:,0], vor.vertices[:,1], 'o')
 
+    line_colors = kw.get('line_colors', 'k')
+    line_width = kw.get('line_width', '1')
+    line_alpha = kw.get('line_alpha', '1.0')
```
Comment (pv) [inline scipy/spatial/_plotutils.py]: Default values for width and alpha probably should be floats, not strings.

**Comment by pv** [comment]
thanks, merged

**Comment by pwolfram** [comment]
Thanks @pv!

**Pull request merged**